exportReport/Export/ExportReport.java

```diff
@@ -0,0 +1,255 @@
+package org.zaproxy.zap.extension.exportReport.Export;
+
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+
+import java.awt.Toolkit;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.io.File;
+import java.io.UnsupportedEncodingException;
+import java.text.MessageFormat;
+import java.util.Locale;
+
+import javax.swing.JOptionPane;
+import javax.swing.SwingWorker;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.extension.ViewDelegate;
+import org.parosproxy.paros.model.Model;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.extension.exportReport.ExtensionExportReport;
+import org.zaproxy.zap.extension.exportReport.FileChooser.*;
+import org.zaproxy.zap.utils.DesktopUtils;
+import org.zaproxy.zap.view.widgets.WritableFileChooser;
+
+/*
+ * AUTHOR		:	GORAN SARENKAPA - JordanGS
+ * SPONSOR		:	RYERSON UNIVERSITY
+ * CLASS		:	ExportReport.java 
+ * DESC			:	FileChooser setup class. Runs the Export as a background task with a Swing Worker.
+ * CREATED ON	:	MARCH 10TH, 2016
+ * CURRENT VER	:	V1.0
+ * SOURCE		:	https://github.com/JordanGS/workspace/tree/master/zap-extensions/src/org/zaproxy/zap/extension/exportReport
+ */
+
+/* 
+ * MODIFED BY	:	<NAME> - <GIT USER>
+ * MOD DATE		:	
+ * MOD VERSION	:	<VERSION OF PLUGIN>
+ * MOD DESC		:	
+ */
```

Comment:
Consider removing these kind of comments, I'm not sure they will be properly maintained (we also have the files versioned for checking the changes).